scheduler/src/cook/kubernetes/api.clj

```diff
@@ -984,7 +992,9 @@
           (.setVolumeMounts container (filterv some? (concat [(init-container-workdir-volume-mount-fn false)
                                                               (scratch-space-volume-mount-fn false)]
                                                              init-container-checkpoint-volume-mounts)))
-          (set-mem-cpu-resources resources total-memory-request total-memory-limit total-cpu-request total-cpu-limit)
+          (if (get (config/kubernetes) :set-memory-limit? true)
```

Comment:
revert this. we actually always want limit for init container and sidecar

Reply:
done